specs/PeriodicBackgoundSyncAndBackgroundSync.md

```diff
@@ -0,0 +1,581 @@
+  ///
+  [propget] HRESULT SyncRegistrationManager([out, retval] ICoreWebView2ServiceWorkerSyncRegistrationManager** value);
+}
+
+/// This is the ICoreWebView2ServiceWorkerSyncRegistrationManager interface for the
+/// Service Worker Periodic Background Synchronization and Background Synchronization
+/// registration management APIs.
+interface ICoreWebView2ServiceWorkerSyncRegistrationManager : IUnknown {
+  /// Add an event handler for the `BackgroundSyncRegistered` event.
+  ///
+  /// \snippet ScenarioSyncRegistrationManager.cpp BackgroundSyncRegistered
+  ///
+  HRESULT add_BackgroundSyncRegistered(
+      [in] ICoreWebView2SyncRegisteredEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_BackgroundSyncRegistered`.
+  HRESULT remove_BackgroundSyncRegistered(
+      [in] EventRegistrationToken token);
+
+  /// Add an event handler for the `PeriodicSyncRegistered` event.
+  ///
+  /// \snippet ScenarioServiceWorkerSyncRegistrationManager.cpp PeriodicSyncRegistered
+  ///
+  HRESULT add_PeriodicSyncRegistered(
+      [in] ICoreWebView2SyncRegisteredEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_PeriodicSyncRegistered`.
+  HRESULT remove_PeriodicSyncRegistered(
+      [in] EventRegistrationToken token);
+
+  /// Gets all background synchronization or periodic background synchronization
+  /// registrations.
+  HRESULT GetSyncRegistrations(
+      [in] COREWEBVIEW2_SERVICE_WORKER_SYNCHRONIZATION_KIND kind
+      , [in] ICoreWebView2ServiceWorkerSyncRegistrationManagerGetSyncRegistrationsCompletedHandler* handler
+  );
+}
+
+/// Event args for the `PeriodicSyncRegistered` event or the `BackgroundSyncRegistered` event.
+[uuid(ff0810cc-a7aa-5149-88cb-2a342233d7bf), object, pointer_default(unique)]
+interface ICoreWebView2ServiceWorkerSyncRegisteredEventArgs : IUnknown {
+  /// The background synchronization task or periodic background synchronization task registration info.
+  ///
+  /// See `ICoreWebView2ServiceWorkerSyncRegistrationInfo` for details on synchronization task registration
+  /// properties.
+  ///
+  [propget] HRESULT RegistrationInfo([out, retval] ICoreWebView2ServiceWorkerSyncRegistrationInfo** value);
+}
+
+/// Provides a set of properties for a service worker synchronization registration.
+[uuid(8ab3dbf2-9207-5231-9241-167e96d8cf56), object, pointer_default(unique)]
+interface ICoreWebView2ServiceWorkerSyncRegistrationInfo : IUnknown {
+  /// The minimum interval time, in milliseconds, at which the minimum time
+  /// interval between periodic background synchronizations should occur.
+  /// You should not call `CoreWebView2ServiceWorkerSyncRegistrationManager.DispatchPeriodicSyncEventAsync`
+  /// to dispatch periodic synchronization tasks less than its minimum time interval.

```

Comment:
Can you say in here why we end devs shouldn't do this?

Reply:
I believe we said in chat that you would add something like 'This is because the periodic sync web APIs standard says it will not be dispatched more often than the minimum time interval and the web content may not handle the event being raised more often.'